docs/build-from-source.md

```diff
@@ -22,7 +22,13 @@ $ cd jw-ott-webapp
 
 ## Install dependencies
 
-Dependencies are installed using Yarn. Use the following command to install all required dependencies. This may take a few minutes.
+Dependencies are installed using Yarn. Use the following command to install required dependencies. This may take a few minutes.
+
+```shell
+$ yarn --ignore-optional
+```
+
+Optional dependencies include packages that aren't necessary to build the project so you can safely ignore them for now. Some of the [easy deployments](./easy-deployments.md) instructions require to install optional dependencies, to install all dependencies including optional dependencies simply exectue the following command. You can run `yarn` at a later time after you already ran `yarn --ignore-optional`.
```

Comment:
Bit of a confusing sentence to me. Here's my suggestions, though Kevin should have final say.
```suggestion
Optional dependencies include packages that aren't necessary to build the project so you can safely ignore them. Some of the [easy deployments](./easy-deployments.md) instructions require installing these optional dependencies.  Use the following command to install all dependencies. This can be run even if you already ran `yarn --ignore-optional`.
```